report_templates/entitlements.erb

```diff
@@ -12,20 +12,15 @@ template_inputs:
   options: "7\r\n30\r\n60\r\n90\r\n120\r\n180"
 require:
 - plugin: katello
-  version: 3.14.0
+  version: 3.16.0
 -%>
-<%- days_from_now = input('Days from Now').to_i -%>
+<%- days_from_now = input('Days from Now') -%>
 <%- should_filter = !input('Days from Now').empty? -%>
-<%- report_headers 'Name', 'Organization', 'Lifecycle Environment', 'Content View', 'Host Collections', 'Virtual', 'Guest of Host', 'OS', 'Arch', 'Sockets', 'RAM', 'Cores', 'SLA', 'Products', 'Subscription Name', 'Subscription Type', 'Subscription Quantity', 'Subscription SKU', 'Subscription Contract', 'Subscription Account', 'Subscription Start', 'Subscription End', 'Subscription Guest', 'Subscription Ends In' -%>
+<%- report_headers 'Name', 'Organization', 'Lifecycle Environment', 'Content View', 'Host Collections', 'Virtual', 'Guest of Host', 'OS', 'Arch', 'Sockets', 'RAM', 'Cores', 'SLA', 'Products', 'Subscription Name', 'Subscription Type', 'Subscription Quantity', 'Subscription SKU', 'Subscription Contract', 'Subscription Account', 'Subscription Start', 'Subscription End', 'Subscription Guest', 'Subscription Expires In' -%>
 <%- load_hosts(includes: [:lifecycle_environment, :operatingsystem, :architecture, :content_view, :organization, :reported_data, :subscription_facet, :pools => [:subscription]], -%>
-<%- search: should_filter && "pools_expiring_in_days = #{days_from_now}"-%>
-<%- ).each_record do |host| -%>
-<%-   if should_filter -%>
-<%-     pools_to_show = host.pools_expiring_in_days(days_from_now) -%>
-<%-   else -%>
-<%-     pools_to_show = host.pools -%>
-<%-   end -%>
-<%-   pools_to_show.each do |pool| -%>
+<%-   search: should_filter && "pools_expiring_in_days = #{days_from_now}" -%>
+<%-   ).each_record do |host| -%>
```

Comment:
this line indentation seems off, sorry if I was not clear in the last comment, this line now needs minus 2 spaces.